packages/react-vapor/src/components/table-hoc/tests/TableWithEmptyState.spec.tsx

```diff
@@ -77,4 +81,29 @@ describe('TableWithEmptyState', () => {
 
         jest.useRealTimers();
     });
+
+    it('should not call the setEmptyState action from the TableHOCActions if the table rendered has the setEmptyState property already set', () => {
+        jest.spyOn<any, string>(TableSelectors, 'isEmptyStateAlreadySet').mockReturnValueOnce(true);
+        const setEmptyStateSpy = jest.spyOn<any, string>(TableHOCActions, 'setEmptyState');
+
+        render(
+            <Provider store={getStoreMock()}>
+                <TableWithEmptyState id="🌶" data={['🤓']} renderBody={() => null} emptyState={<EmptyState />} />
+            </Provider>
+        );
```

Comment:
render function already wraps everything inside a store provider

```suggestion
        render(
                <TableWithEmptyState id="🌶" data={['🤓']} renderBody={() => null} emptyState={<EmptyState />} />
        );
```